cross_list_courses/templates/list.html

```diff
@@ -8,6 +8,11 @@ <h3>
         Cross Listings
     </h3>
     </nav>
+     <div class="row">
```

Comment:
I believe this row element should be placed in the "container-fluid" div below. Otherwise the row style of ".row {
    margin-right: -15px;
    margin-left: -15px;
}" creates horizontal scrolling on the page.